Gemfile.lock

```diff
@@ -37,10 +37,8 @@ GEM
     loofah (2.9.0)
       crass (~> 1.0.2)
       nokogiri (>= 1.5.9)
-    mini_portile2 (2.5.0)
-    minitest (5.14.3)
-    nokogiri (1.11.1)
-      mini_portile2 (~> 2.5.0)
+    minitest (5.14.4)
+    nokogiri (1.11.1-x86_64-darwin)
```

Comment:
On the line 39, there is the same Library with different version. Does it work for multi Environments or we have to have those 2?

Reply:
This is the actual dependency installed: nokogiri (1.11.1-x86_64-darwin). The other 2 lines are nested under another library and have a range of acceptable versions. Line 39 has this: `nokogiri (>= 1.5.9)` which means that the `loofah` gem accepts any nokogiri version equal to or greater than 1.5.9.  In this case we have 1.11.1, which means it falls within the acceptable range.